administration/configuring-fluent-bit/classic-mode/variables.md

```diff
@@ -1,6 +1,7 @@
 # Variables
 
 Fluent Bit supports the usage of environment variables in any value associated to a key when using a configuration file.
+<img referrerpolicy="no-referrer-when-downgrade" src="https://static.scarf.sh/a.png?x-pxid=1731c7b5-34c6-424f-bfc6-88c2aa71e81f" />
```

Comment:
Same here. Since it was h1, blank line, scarf image, blank line, and then content, we should do that here as well.